en/docs/page-not-found.md

```diff
@@ -23,6 +23,6 @@ template: templates/no-nav.html
 ## Page not found
 
 
-Sorry but the page you are looking for does not exist, has been removed, the name changed, or is temporarily unavailable.   
-For inquiries, please reach us at [dev@wso2.org](mailto:dev@wso2.org). 
+Sorry but the page you are looking for does not exist, has been removed, changed, or is temporarily unavailable.   
+For inquiries, please reach us at [documentation@wso2.com](mailto:documentation@wso2.com). 
```

Comment:
Let's ask to report a GitHub issue(https://github.com/wso2/docs-is/issues) or send mail to the public mailing list dev@wso2.org.

Reply:
+1